src/wallet/test/fuzz/coinselection.cpp

```diff
@@ -77,7 +77,7 @@ FUZZ_TARGET(coinselection)
     const CFeeRate effective_fee_rate{ConsumeMoney(fuzzed_data_provider, /*max=*/COIN)};
     const CAmount cost_of_change{ConsumeMoney(fuzzed_data_provider, /*max=*/COIN)};
     const CAmount min_viable_change{ConsumeMoney(fuzzed_data_provider, /*max=*/COIN)};
-    const CAmount target{fuzzed_data_provider.ConsumeIntegralInRange<CAmount>(1, MAX_MONEY)};
+    const CAmount target{fuzzed_data_provider.ConsumeIntegralInRange<CAmount>(50000, MAX_MONEY)};
```

Comment:
what was the reason behind this change?

Reply:
just changed to `1000`. but the reason is to make it more realistic. e.g. do we really use it for 1 sat?